content/en/security_platform/application_security/how-appsec-works.md

```diff
@@ -27,13 +27,22 @@ Datadog ASM identifies bad actors by collecting client IP addresses and manually
 
 ## Identify services exposed to application attacks
 
-Datadog ASM uses the information APM is already collecting, and flags traces containing attack attempts. Services exposed to application attacks are highlighted directly in the security views embedded in APM ([Service Catalog][14], [Service Page][15], [Traces][16]).
+Datadog ASM uses the information APM is already collecting, and flags traces containing attack attempts. Services exposed to application attacks are highlighted directly in the security views embedded in APM ([Service Catalog][1], [Service Page][2], [Traces][3]).
 
 Because APM collects a sample of your application traffic, enabling ASM in the tracing library is necessary to effectively monitor and protect your services.
 
 ## Compatibility
 
-For Datadog ASM to be compatible with your Datadog configuration, you must have APM enabled, and [send traces to Datadog][1]. ASM uses the same libraries used by APM, so you don't need to deploy and maintain another library. Steps to enable Datadog ASM are specific to runtime language. Check to see if your language is supported in the [ASM prerequisites][2].
+For Datadog ASM to be compatible with your Datadog configuration, you must have APM enabled, and [send traces to Datadog][4]. ASM uses the same libraries used by APM, so you don't need to deploy and maintain another library. Steps to enable Datadog ASM are specific to runtime language. Check to see if your language is supported in the [ASM prerequisites][5].
+
+### Serverless monitoring
+
+<div class="alert alert-info">Serverless threat monitoring is in private beta. Request to join the early preview with <a href="https://docs.google.com/forms/d/e/1FAIpQLScB3uSccf9lSAh7GcA8NZ8SsmUGQ5mi09DnDgZmqXcbiYfMzA/viewform">this form</a>.</div>
+ 
+Datadog ASM supports functions deployed on AWS Lambda. Detection is done by using the [Lambda extension][6]. 
+
+Full threat monitoring capabilities are available for Lambda functions. You can detect attackers targeting your functions, trace their attack flow with deep network and code-level insights, and remediate the threat.
```

Comment:
I updated the wording slightly here, how does this sound? 

"Full threat monitoring capabilities are available for Lambda functions. You can detect attackers targeting your functions, trace their attack path with deep code level insights, then finally remediate the threat. "

Reply:
Fixed the grammar (then is not a conjunction), but otherwise lgtm